pytest_fixtures/component/provision_azure.py

```diff
@@ -40,7 +39,9 @@ def sat_azure_domain(sat_azure, sat_azure_loc, sat_azure_org):
 def sat_azure_default_os(sat_azure):
     """Default OS on the Satellite"""
     return (
-        sat_azure.api.OperatingSystem().search(query={'search': DEFAULT_OS_SEARCH_QUERY})[0].read()
+        sat_azure.api.OperatingSystem()
+        .search(query={'search': settings.DEFAULT_OS_SEARCH_QUERY})[0]
```

Comment:
Caught by PRT: should be `settings.server.default_os_search_query`.
Does this work for you, dosas? I'd almost say this can't work.